### PR TITLE
app-backup/deja-dup: version bump to 43.4

### PR DESCRIPTION
**Pull request by Schievel1**
Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>

### CODE REVIEW

**Comment by Schievel1** [comment]
Oh I accidentally did this PR because I assumed this is maintained by gnome project.
I wrote the maintainer a mail to check if he wants to use this PR to bump the package

**Comment by Schievel1** [comment]
@oz123 says he is okay with this PR

**Comment by oz123** [comment]
Yes, I'm fine .

**Comment by Schievel1** [comment]
Yes I know I misspelled that, will correct later

**Comment by mattst88** [comment]
Please add a package.mask entry:



  | NonsolvableDepsInDev | nonsolvable depset(rdepend) keyword(~amd64) dev profile (default/linux/amd64/17.0/x32) (7 total): solutions: [ >=net-libs/libsoup-3.0:3 ]
-- | -- | --
  | NonsolvableDepsInStable | nonsolvable depset(depend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (14 total): solutions: [ >=net-libs/libsoup-3.0:3 ]

File: app-backup/deja-dup/deja-dup-43.4.ebuild [hunk 1]
```diff
@@ -24,16 +24,19 @@ BDEPEND="
...
+	>=gui-libs/gtk-4.6:4
 	>=dev-libs/json-glib-1.2
-	dev-libs/libgpg-error
-	>=gui-libs/libhandy-1.0:1
-	>=net-libs/libsoup-2.48:2.4
-	>=x11-libs/gtk+-3.22:3
-	>=x11-libs/libnotify-0.7
+	>=dev-libs/glib-2.64:2[dbus]
+	>=app-crypt/libsecret-0.18.6[vala]
+	>=net-libs/libsoup-3.0:3
```
Comment (mattst88) [inline app-backup/deja-dup/deja-dup-43.4.ebuild:37]: `:3.0`

File: profiles/package.mask [hunk 2]
```diff
@@ -438,6 +438,7 @@ net-libs/webkit-gtk:4.1
 sci-geosciences/geocode-glib:2
 >=sci-geosciences/gnome-maps-43
 >=x11-wm/mutter-43
+>=app-backup/deja-dup-43.1
```
Comment (mattst88) [inline profiles/package.mask:441]: This is an alphabetized list.

File: app-backup/deja-dup/Manifest [hunk 3]
```diff
@@ -1,2 +1,2 @@
-DIST deja-dup-42.7.tar.bz2 1384797 BLAKE2B f462df835039606b721d935e086e51ee27e861baa1fbc5836bb79952c838921cf5f28d439465451c5259bc2c77fa9cb839488c8b0196c6d176e32d9e42180539 SHA512 80c57fe75635346cf2597195e7f7b76e2562aaa6900e78fbfb7ad6ada9c268df36d4e16aa7ca522e1570987f5826586e1cab5fecb7d5ba90e34a23a349b47a74
```
Comment (mattst88) [inline app-backup/deja-dup/Manifest:1]: This is in the wrong commit.

**Review by mattst88 — APPROVED** [review]
I'll fix the things I pointed out, but really -- you've gotta review your own commits for mistakes like these. Plea se run `pkgcheck scan --commits` and fix warnings it gives.

**Comment by mattst88** [comment]
Also, commit titles should be in the imperative mood. E.g. "Add xyz" rather than "Added xyz".

**Pull request closed without merging**